packages/apps/desktop-viewer-test/src/frontend/components/App.tsx

```diff
@@ -23,7 +23,12 @@ import {
 } from "./routes";
 
 const App = () => {
-  const initialized = useDesktopViewerInitializer();
+  (window as any).ITWIN_VIEWER_HOME = window.location.origin;
+
+  const initialized = useDesktopViewerInitializer({
```

Comment:
Why do we need additional names pace? Previously we made use of the itwinviewer ns that is registered in the base package.  Not possible?

Reply:
Tbh not sure how localization worked in the previous gen or if it worked at all. There are strings shown on the home route that are defined in a   json file in the desktop app, but had the same file name as the one in the base package so there was a collision there, and it doesn't seem to merge them so we only get strings from the base viewer and not the desktop app. 

I thought this approach was better than moving the other strings into the base because now users can add their app strings too.